src/plugins/saved_objects_management/public/lib/import_file.ts

```diff
@@ -40,12 +40,12 @@ interface ImportResponse {
 export async function importFile(
   http: HttpStart,
   file: File,
-  { createNewCopies, overwrite }: ImportMode,
+  { createNewCopies, overwrite, workspaces }: ImportMode,
   selectedDataSourceId?: string
 ) {
   const formData = new FormData();
   formData.append('file', file);
-  const query = createNewCopies ? { createNewCopies } : { overwrite };
+  const query = createNewCopies ? { createNewCopies, workspaces } : { overwrite, workspaces };
```

Comment:
> do we have tests for the /_import endpoint with workspaces?

Yes, we have a integration test for importing with workspaces.

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/workspace/server/saved_objects/integration_tests/saved_objects_wrapper_for_check_workspace_conflict.test.ts#L257